kik_unofficial/simple.py

```diff
@@ -0,0 +1,12 @@
+import logging
```

Comment:
I don't like this file named `simple.py`. let's move this into a function called `set_up_basic_logging` or so under `utilities\logging_utils.py`.